stdlib/stdlib.mli

```diff
@@ -1060,16 +1060,29 @@ val input : in_channel -> bytes -> int -> int -> int
    [input] must be called again to read the remaining characters,
    if desired.  (See also {!Stdlib.really_input} for reading
    exactly [len] characters.)
-   Exception [Invalid_argument "input"] is raised if [pos] and [len]
+   The exception [Invalid_argument "input"] is raised if [pos] and [len]
```

Comment:
Why not use `@raise`?

Reply:
I just reused the existing documentation comments. I may change the three I touched (input, really_input, really_input_up_to) to use `@raise`.

Reply:
Done.